### PR TITLE
Sync changes from zowe.org

### PR DESCRIPTION
**Pull request by nannanli**
Signed-off-by: nannanli <nannanli@cn.ibm.com>

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -2,27 +2,32 @@
...
-Zowe consists of the following main components.
 
-<p><b>zLUX:</b> Contains a Web user interface (UI) that provides a full screen interactive experience. The Web UI includes many interactions that exist in 3270 terminals and web interfaces such as IBM z/OSMF. </p>
+<p>Zowe consists of the following main components.</p>
 
-<p><b>Explorer server:</b> Provides a range of APIs for the management of jobs, data sets, z/OS UNIX System Services files, and persistent data. </p>
+<p><b>Zowe Application Framework:</b> A web user interface (UI) that provides a virtual desktop containing a number of apps allowing access to z/OS function.  Base Zowe includes apps for traditional access such as a 3270 terminal and a VT Terminal, as well as an editor and explorers for working with JES, MVS Data Sets and Unix System Services.</p>
 
-<p><b>API Mediation Layer:</b> Provides an API abstraction layer through which APIs can be discovered, catalogued, and presented uniformly. </p>
+<p><b>z/OS Services:</b> Provides a range of APIs for the management of z/OS JES jobs and MVS data set services.</p>
```
Comment (jmertic) [inline index.md]: My only concern here is that the other items are matching to Zowe sub-projects, but this one isn't. Not sure....

Comment (nannanli) [reply index.md]: Hi John, I removed the z/OS Services from the component list. It is documented in the doc site as one of the four components. However I noticed that it's not on zowe.org now. I guess someone removed it at some time for certain reasons. I'll double confirm with folks to see if this is required. Thanks!

File: index.md [hunk 2]
```diff
@@ -34,32 +37,42 @@ Zowe consists of the following main components.
 
 <section class="bluebackground">
 
-<h1 id="download">DOWNLOAD</h1>
+<h1 id="documentation">Try Zowe now</h1>
+<p>
+Try Zowe today at no charge, and with no installation required. Your no-charge trial is available in two hours for three days. Register and get started today. 
+</p>
+<button><a href="https://www.ibm.com/account/reg/signup?formid=urx-38870">Request a trial</a></button>
+
```
Comment (jmertic) [inline index.md]: Can we add some verbiage under the link like...

"By Proceeding to the trial, you will be leaving the zowe.org website."

Also, can you change the link to https://www.openmainframeproject.org/projects/zowe/ztrial ( which will forward to the same URL ).

Comment (nannanli) [reply index.md]: The requested updates are made. : )

**Pull request merged**